druid-shell/src/platform/windows/window.rs

```diff
@@ -595,6 +613,29 @@ impl WndProc for MyWndProc {
                     let s = s.as_mut().unwrap();
                     let x = LOWORD(lparam as u32) as i16 as i32;
                     let y = HIWORD(lparam as u32) as i16 as i32;
+
+                    // When the mouse first enters the window client rect we need to register for the
+                    // WM_MOUSELEAVE event. Note that WM_MOUSEMOVE is called even when the change
+                    // window under the cursor changes without moving the mouse, for example when
```

Comment:
Is this an accidental extra word? *when the ~change~ window under the cursor changes*

Reply:
You're right, fixed